SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -65,6 +65,71 @@ In this section you’ll run a [local deployment](https://docs.improbable.io/ref
 3. Back in your two Unreal game clients, run around and shoot.
 4. Using the Inspector to track the location of your two players, notice that if you position them in the area of authority then their shots damage each other, but if they are on different servers, they can’t damage each other. Let’s fix that.
 
+### Replicate health changes
+
+In the ThirdPersonShooter project each `TPSCharacter` has a variable tracking health called `CurrentHealth`. The health deducted on the server when the character is being shot, however as the variable is not setup for replication it does not replicate to the client. Therefore the characters health meter does not update on the client even though the health variable is being depleted on the server. 
+
+To resolve this we need to mark the `CurrentHealth` property for replication as you would with normal [Unreal Actor replication](https://docs.unrealengine.com/en-us/Resources/ContentExamples/Networking/1_1):
+
+Navigate to the declaration of the `CurrentHealth` variable in the `TPSCharacter.h`(line 175) and change the line:
+```
+UPROPERTY()
+int32 CurrentHealth;
+```
+To
+```
+UPROPERTY(ReplicatedUsing = OnRep_CurrentHealth)
+int32 CurrentHealth; 
+```
+
+We have now marked this property for replication using the `OnRep_CurrentHealth` function which we’ll implement in a moment.
+
+Next we need to update the [GetLifetimeReplicatedProps](https://wiki.unrealengine.com/Replication#Actor_Property_Replication) implementation of the TPSCharacter to specify the [replication conditions](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties/Conditions) for the `CurrentHealth` variable:
+
+Navigate to `TPSCharacter.cpp` line 194 and add the following lines:
```

Comment:
Perhaps we should specify more detail here rather than just the line number.

Reply:
Yes that will be for the tidied up tutorial later